figures/mapping-histology-labels.Rmd

```diff
@@ -172,6 +172,28 @@ if (nrow(display_group_df) > 17) {
 }
 ```
 
+# Make `display_order`
+
+Get ranks in order of big to small and make them into a new column in `display_group_df`. 
+We will always want the "normal", "benign", "other_tumor" groups to come last so we are gonna remove them from this ranking and then add back when we get to `histology_table`. 
+
+```{r}
+control_like_groups <- c("normal", "benign", "other tumor")
+
```

Comment:
I don't think you use this again?
```suggestion
```

Reply:
Oh yeah, I used it before but don't know.